client/globals.d.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Internal dependencies
  */
-import {
+import type {
```

Comment:
I think I asked you this but forgot the reason.

Googled and found https://medium.com/@quizzesforyou/import-vs-import-type-in-typescript-8e5177b62bea

Reply:
This is my personal preference, I think it is more explicit === improved code readability.